src/components/Popover/Popover.module.css

```diff
@@ -0,0 +1,162 @@
+@import '../../design-tokens/mixins.css';
+
+/*------------------------------------*\
+    # POPOVER
+\*------------------------------------*/
+
+/**
+ * Popover window
+ * 1) Actual box with content within
+ */
+.popover {
+  display: flex;
+  flex-direction: column;
+  position: absolute;
+  visibility: hidden;
```

Comment:
Just curious - is there any particular reason you're using `visibility: hidden;` instead of `display: none;`? Is there an advantage?